lib/rouge/lexers/gdscript.rb

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class GDScript < RegexLexer
+      title "GDScript"
+      desc "The Godot Engine programming language (https://godotengine.org/)"
+      tag 'gdscript'
+      aliases 'gd', 'gdscript'
+      filenames '*.gd'
+      mimetypes 'text/x-gdscript', 'application/x-gdscript'
+
+      def self.keywords
+        @keywords = %w(
+          and in not or as breakpoint class class_name extends is func setget
+          signal tool const enum export onready static var break continue
+          if elif else for pass return match while remote master puppet
+          remotesync mastersync puppetsync
+        ).join('|')
+      end
+
+      # Reserved for future implementation
+      def self.keywords_reserved
+        @keywords_reserved = %w(
+          do switch case
+        ).join('|')
+      end
+
+      def self.builtins
+        builtins = %w(
+          Color8 ColorN abs acos asin assert atan atan2 bytes2var ceil char
+          clamp convert cos cosh db2linear decimals dectime deg2rad dict2inst
+          ease exp floor fmod fposmod funcref hash inst2dict instance_from_id
+          is_inf is_nan lerp linear2db load log max min nearest_po2 pow
+          preload print print_stack printerr printraw prints printt rad2deg
+          rand_range rand_seed randf randi randomize range round seed sign
+          sin sinh sqrt stepify str str2var tan tan tanh type_exist typeof
+          var2bytes var2str weakref yield
+        ).join('|')
+      end
+
+      def self.builtins_type
+        @builtins_type = %w(
+          bool int float String Vector2 Rect2 Transform2D Vector3 AABB
+          Plane Quat Basis Transform Color RID Object NodePath Dictionary
+          Array PoolByteArray PoolIntArray PoolRealArray PoolStringArray
+          PoolVector2Array PoolVector3Array PoolColorArray null
+        ).join('|')
+      end
+
+      state :root do
+        rule %r/\n/, Text
+        rule %r/[^\S\n]+/, Text
+        rule %r/#.*/, Comment::Single
+        rule %r/[\[\]{}:(),;]/, Punctuation
+        rule %r/(in|and|or|not)\b/, Operator::Word
+        rule %r/!=|==|<<|>>|&&|\+=|-=|\*=|\/=|%=|&=|\|=|\|\||[-~+\/\\*%=<>&^.!|$]/, Operator
+        rule %r/(func)((?:\s|\\)+)/ do
+          groups Keyword, Text
+          push :funcname
+        end
+        rule %r/(class)((?:\s|\\)+)/ do
+          groups Keyword, Text
+          push :classname
+        end
+        mixin :keywords
+        mixin :builtins
+        rule %r/(""")/ do
+          groups Str::Affix, Str::Double
```

Comment:
You've removed the interpolation of `affix_long` from the rule but you're still matching multiple groups (which you no longer have). If you do leave out the interpolate, you can express this rule as:

```ruby
rule %r/"""/, Str::Double, :escape_tdqs
```

Similar changes could be made for the other strings.